text-classifier/SigOpt Text Classifier Walkthrough.ipynb

```diff
@@ -25,18 +25,7 @@
    "metadata": {
     "collapsed": false
    },
-   "outputs": [
-    {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "POSITIVE: My husband is a BMW mechanic and he drives cars all day.  I wish I could get him one for every car\n",
-      "\n",
-      "NEGATIVE: Far better models for similair price range - dissapointed with the quality of the finish etc etc. works fine though.\n",
-      "\n"
-     ]
-    }
-   ],
+   "outputs": [],
```

Comment:
Ah ok interesting. Hmmm. This caught my attention because of "View your experiment details at https://sigopt.com/experiment/2309\n" which of course isn't possible for the casual reader. Scott's Vegas post only includes outputs where there is a graph or something that you should expect to see.

How about we just remove the results page links and leave in the rest of the output?